Jellyfin.Plugin.Bookshelf/Jellyfin.Plugin.Bookshelf.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Jellyfin.Plugin.Bookshelf</RootNamespace>
+    <AssemblyVersion>1.0.0</AssemblyVersion>
+    <FileVersion>1.0.0</FileVersion>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Jellyfin.Common" Version="10.*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.*" />
```

Comment:
IIRC this is the only package reference actually needed as it depends on the other 2.